commisery/rules.py

```diff
@@ -482,6 +482,7 @@ def validate_commit_message_rule(
             get_default_rules()[rule].get("obj")(message, config)
     except logging.Error as err:
         err.message = f"[{rule}] {err.message}"
+        err.file_path = message.hexsha
```

Comment:
Worthwhile to note that the line-number data is already inserted in all rules, but did not show up as no `file_path` was specified